evap/staff/importers.py

```diff
@@ -3,7 +3,7 @@
 
 from django.conf import settings
 from django.db import transaction
-from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext as _, ugettext_lazy as _lazy
```

Comment:
Thanks for contributing! :)

This is just a minor thing, but I'd rather think of `_` as the only shorthand and not create the similar shorthand `_lazy`. Thus, I'd ask you to just
```
from django.utils.translation import ugettext_lazy, ugettext as _
```
and use `ugettext_lazy` below.

I've talked with @janno42 and he agreed.
@karyon what do you think?

Reply:
👍

Reply:
done